src/main/java/org/jenkins/plugins/lockableresources/actions/LockableResourcesRootAction.java

```diff
@@ -157,22 +196,32 @@ public int getNumberOfAllLabels() {
    */
   @Restricted(NoExternalUse.class)
   public int getAssignedResourceAmount(String label) {
```

Comment:
check where it is used and set it to deprecated
This operation will be very slowly (because of synchronized over all resources)